CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
+-`ZkProgram` to support non-pure provable types as inputs and outputs https://github.com/o1-labs/o1js/pull/1828
```

Comment:
I thought this should have to go inside the `Unreleased` section?

Reply:
Good catch. Moved it.